tinkerbell/boots/templates/deployment.yaml

```diff
@@ -29,26 +35,44 @@ spec:
       containers:
         - image: {{ .Values.image }}
           imagePullPolicy: {{ .Values.imagePullPolicy }}
-          {{- if .Values.args }}
-          args: 
-          {{- range .Values.args }}
+          args:
+            - -dhcp-addr={{ printf "%v:%v" .Values.dhcp.ip .Values.dhcp.port }}
+            - -ipxe-tftp-addr={{ printf "%v:%v" .Values.tftp.ip .Values.tftp.port }}
+            - -http-addr={{ printf "%v:%v" .Values.http.ip .Values.http.port }}
+            - -syslog-addr={{ printf "%v:%v" .Values.syslog.ip .Values.syslog.port }}
+            - -ipxe-remote-tftp-addr={{ printf "%v:%v" $remoteTftpIp .Values.remoteTftp.port }}
+            - -ipxe-remote-http-addr={{ printf "%v:%v" $remoteHttpIp .Values.remoteHttp.port }}
```

Comment:
These 2 are only used when Boots is not serving the iPXE binaries. I get an error when deploying. `panic: invalid IP for remote TFTP server: 192.168.2.111:69`

Reply:
Thanks. This should be removed now.